### PR TITLE
Split packaging for Clrcompression by appx/non-appx

### PR DESCRIPTION
**Pull request by ianhays**
@weshaggard @ericstj

### CODE REVIEW

File: src/Native/pkg/runtime.native.System.IO.Compression/win7/runtime.native.System.IO.Compression.pkgproj [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+    <PropertyGroup>
+        <Version>4.0.1</Version>
+        <PackageTargetRuntime>win7-$(PackagePlatform)</PackageTargetRuntime>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <File Include="$(WinNativePath)ProjectK\Runtime\clrcompression.dll">
```
Comment (weshaggard) [inline src/Native/pkg/runtime.native.System.IO.Compression/win7/runtime.native.System.IO.Compression.pkgproj]: You should remove "ProjectK\Runtime" from this now. Please do verify that in TFS builds.

File: src/Native/pkg/runtime.native.System.IO.Compression/runtime.native.System.IO.Compression.builds [hunk 2]
```diff
@@ -23,15 +23,15 @@
       <OSGroup>OSX</OSGroup>
       <Platform>amd64</Platform>
     </Project>
-    <Project Include="win\runtime.native.System.IO.Compression.pkgproj">
+    <Project Include="win7\runtime.native.System.IO.Compression.pkgproj">
```
Comment (weshaggard) [inline src/Native/pkg/runtime.native.System.IO.Compression/runtime.native.System.IO.Compression.builds]: Add win10 entries in the builds file as well so those configurations are built.

**Comment by mellinoe** [comment]
LGTM

**Comment by ianhays** [comment]
Huh, can't merge on mobile until the ci finishes. Well this should be good to to, I modeled the paths after what sni did (i.e. winnativepath in tfs goes to projects/native/).

The binaries directory wasn't working when I tested it, but I can't see why. It was linking it to "bin/corefx/open' instead of "projects/runtime/"

**Comment by mellinoe** [comment]
This needs to be rebased, but I can't push to Ian's remote, so I'll just put up a new PR.

**Pull request closed without merging**